ivy/__init__.py

```diff
@@ -599,244 +599,8 @@ class Node(str):
 invalid_uint_dtypes = ()
 invalid_complex_dtypes = ()
 
-# data type promotion
```

Comment:
Can we please move the updated promotion tables to this line? A bit OCD but would be easier to come back to these changes if only the updated parts are tracked by git (`array_api_promotion_table` is still the same)